src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -616,7 +618,10 @@ namespace winrt::Microsoft::Terminal::Control::implementation
     {
         if (_searchBox && _searchBox->Visibility() == Visibility::Visible)

```

Comment:
This one doesn't use `IsOpen()`?

Reply:
I likely have missed that one.. but this is less likely to cause an issue because `_SearchChanged` is invoked for searchbox's TextChanged event. The first thing we do before closing is move focus back to the terminal, so you can't _really_ still be typing more characters into the searchbox after you've pressed <kbd>ESC</kbd>. (_Unless you're Flash and could move the focus back to SB and start typing again.. all while it's closing_  😅 )

Reply:
What would you suggest we do when `GoForwardButton` or `GoBackwardButton` button is clicked when SB is closing? Should we ignore the click or re-open SB? 🤔

Reply:
> Should we ignore the click or re-open SB? 🤔

I'd ignore it as well.